README.md

```diff
@@ -1,6 +1,15 @@
 # Nest.js GCP Logger
 
 
+
+### Module Parameters
+
+| Property              | Default | Description |
+| ---                   | ---     | --- |
+| `GCP_ERROR_REPORTING` | `false` | If set to `true` the `error`s are reognized by GCP Error Reporting by wrapping the provided message in a stack trace |
```

Comment:
* typo recognized 
* maybe formulate a bit more explicit "all error messages are ...", as the term error can be confused with native JS errors